lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -238,7 +238,7 @@ def fetch
                 rename_repos_owner(user.login, info['login'])
                 user.update_attributes info
               else
-                self.user = ::User.create! info
+                self.user = ::User.create! info if Travis::Enterprise.check_license_seat?
```

Comment:
@Tahsin-travis-ci, if check returns false (on hosted) - user will not be created at all? 
UPD: That's a bit confusing, we shouldn't call `check_license_seat?` at all unless it's enterprise.